proteus/tests/post_processing/import_modules/bdm_tests_3Dtemplate.py

```diff
@@ -3,16 +3,19 @@
 from proteus import default_n as n
 from proteus import default_s,default_so
 from proteus import Context
-import numpy, os
+import numpy, os, sys, inspect
```

Comment:
Is the `sys` module necessary here? I don't see it being used in the rest of the file.